mirkowelt: to expose that error ust end your chain with '.end()'